src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -50,7 +50,7 @@ public function __construct($debug = true, $fileLinkFormat = null)
      */
     public static function register($debug = true, $fileLinkFormat = null)
     {
-        $handler = new static($debug, $fileLinkFormat = null);
+        $handler = new static($debug, $fileLinkFormat);
```

Comment:
this is not a dead code removal. It is a bug fix. So it should go in older versions